test-cloud/test-cloud-remote/src/test/resources/deployments/ldap.yaml

```diff
@@ -0,0 +1,142 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: ldap
+  annotations:
+    name: ldap
+    description: Application template for ldap server
+message: A new ldap server was created
+
+parameters:
+- name: APPLICATION_NAME
+  displayName: Application Name
+  description: The name for the application.
+  required: true
+- name: LDAP_DOCKER_IMAGE
+  displayName: LDAP Docker Image
+  description: The LDAP Docker Image URL
+  required: true
+- name: LDAP_USER_NAME
+  displayName: LDAP User Name
+  required: true
+- name: LDAP_USER_PWD
+  displayName: LDAP User Password
+  required: true
+objects:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: "rhba-qe-ldap"
+    annotations:
+      openshift.io/image.insecureRepository: 'true'
+  spec:
+    tags:
+    - name: 'latest'
+      from:
+        kind: DockerImage
+        name: ${LDAP_DOCKER_IMAGE}
+      importPolicy:
+        insecure: true
+      referencePolicy:
+        type: Local
+- kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: ldap-user-scripts
+  data: 
+    addUser.sh: |- 
+      ldapadd -h ldap-${APPLICATION_NAME} -p 30389 -x -D cn=Manager,dc=example,dc=com -w admin << EOF
```

Comment:
I added this script so we ensure that the user we specify in the test params, has the right credentials in LDAP.